vscode-extension/src/welcomePage.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
```

Comment:
Why isn't this markdown?

Reply:
Yes you can open the readme in preview mode, we don't need to open it as a webview.

Reply:
You should be able to do something like this:

```
function activate(context) {
    let previousVersion = context.globalState.get('version');
    let currentVersion = vscode.extensions.getExtension(EXTENSION_NAME).packageJSON.version;

    if(previousVersion === undefined || previousVersion !== currentVersion) {
        let readmePath = path.join(context.extensionPath, 'README.md');
        let readme = fs.readFileSync(readmePath, 'utf8');
        // Open raw markdown (not needed)
        //let document = await vscode.workspace.openTextDocument({ content: readme, language: 'markdown' });
        //vscode.window.showTextDocument(document);

        // Open markdown in preview mode
        vscode.commands.executeCommand('markdown.showPreview', vscode.Uri.file(readmePath))
    
        // Update the version in the globalState
        context.globalState.update('version', currentVersion);
    }
}
```

The code inside the IF block can be refactored into the WELCOME command

Reply:
Also we should have an information message that pops up when the Welcome page is shown. Can also have a setting called 'showWelcome' which can be set to false if a user selects "Don't Show Again" in that information message. Can happen at the top of the stack.